operator/apis/machinelearning.seldon.io/v1/conversion.go

```diff
@@ -0,0 +1,146 @@
+package v1
+
+import (
+	autoscalingv2 "k8s.io/api/autoscaling/v2"
+	autoscalingv2beta1 "k8s.io/api/autoscaling/v2beta1"
```

Comment:
Does it mean that our logic here will work as long as we rely on version of K8s client that still comes with `k8s.io/api/autoscaling/v2beta1`? And once we upgrade we will have to bring in the relevant `v2beta1` code into the repo/.

Reply:
yes